core/tests/protractor_desktop/topicAndStoryEditor.js

```diff
@@ -411,14 +411,10 @@ describe('Chapter editor functionality', function() {
   it('should fail to add one prerequisite skill which is already added as' +
     ' acquired skill', async function() {
     await storyEditorPage.navigateToChapterWithName('Chapter 1');
-    await storyEditorPage.addAcquiredSkill(dummySkills[1]);
+    await storyEditorPage.addPrerequisiteSkill(dummySkills[0]);
```

Comment:
As the test description suggests we should not be able to add a prerequisite skill but previously we were adding an acquired skill.
Now dummySkills[0] is already added as acquired skills in the previous tests, so we should not be able to add it again as a prerequisite skill.